backend/dal/Migrations/v01.05.06/Up/PostUp/01-RoleClaims.sql

```diff
@@ -0,0 +1,54 @@
+PRINT 'Adding Role'
```

Comment:
You will need to do one of the following in the case of a rollback.

- Undo any of the changes you made here
- or, update this script to only perform these updates if they haven't already been done